app/src/main/java/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -547,4 +547,13 @@ private boolean allNotFavorites() {
         }
         return true;
     }
+
+    private boolean isShared() {
+        for (OCFile file : files) {
+            if (file.isSharedViaLink() || file.isSharedWithSharee()) {
```

Comment:
Shouldn't this also check `file.isSharedWithMe()`?

```suggestion
            if (file.isSharedWithMe() || file.isSharedViaLink() || file.isSharedWithSharee()) {
```

Reply:
Of course!